pg: add general IOP limit information to devportal (#497)
aiven/devportal: docs/products/postgresql/concepts/check-iops.rst

IOPS capability of plans
========================

> This article will explain what is the typical IOPS capability used for common plans in pg:

Reviewer: Can we add a bit more description here? Like
* explaining the IOPS acronym and why one should be interested in knowing more
* instead of pg we should write Aiven for PostgreSQL

    * - **startup-120**
      - ``i3.4xlarge``           
      - custom type             
      - ``standard_l16s_v2``
    * - **startup-190**
      - ``m5.12xlarge``          
      - custom type             
      - not available
    * - **startup-240**
      - ``i3.8xlarge``
      - ``n1-standard-64``          
      - ``standard_l32s_v2``
    * - **startup-360**
      - ``m5ad.24xlarge``        
      - ``n2d-standard-96``         
      - not available
    * - **startup-512**                      
      - not available        
      - ``n2d-standard-128``        
      - not available
    * - **startup-512 compute-optimized**   
      - ``m5.2xlarge``           
      - not available           
      - not available
    * - **startup-512 storage-optimized**      
      - ``m5.large``             
      - not available           
      - not available
    * - **startup-512 io-optimized**          
      - ``i3.large``             
      - not available           
      - not available
    * - **startup-896**                        
      - not available        
      - ``n2d-standard-224``        
      - not available

Business and premium
'''''''''''''''''''''

> The VM type is similar as startup while with 2 and 3 node high availability set.

Reviewer: Can we add a bit more words and try avoiding acronyms? Like 
```suggestion
The the virtual machine types in the Business and premium plans are equal to the startup ones. The difference is the cluster size with 2 notes for Business plans and 3 node high availability set for premium plans.
```

> Node capacity

Reviewer: ```suggestion
Single node capacity
```

'''''''''''''

> The IOPS limitation can be found as per:

Reviewer: ```suggestion
The single node IOPS limitation detail can be found in the cloud vendor dedicated documentation:
```

AWS: https://aws.amazon.com/ec2/instance-types/

GCP: https://cloud.google.com/compute/docs/general-purpose-machines

Azure: https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-previous-gen

> While the instance type may vary because of different regions to another equivalent type.

Reviewer: ```suggestion
The instance types can vary to other equivalent types in different regions .
```

> .. note:: For custom plans, if you want to know the node types for IOPS limitation, please contact support team.

Reviewer: how should someone contact the support team? Should we add an email address here?
